spec/search_builders/curation_concerns/admin_set_search_builder_spec.rb

```diff
@@ -9,7 +9,7 @@
     before { builder.filter_models(solr_params) }
 
     it 'adds AdminSet to query' do
-      expect(solr_params[:fq].first).to include('{!field f=has_model_ssim}AdminSet')
```

Comment:
My confusion comes from the spec saying "field" and the above `filter_by_type.rb` has "terms" and the "terms" hasn't changed from "field".